lib/ruby_lsp/ruby_document.rb

```diff
@@ -158,11 +158,16 @@ def parse!
       return false unless @needs_parsing
 
       @needs_parsing = false
-      @parse_result = Prism.parse(@source)
+      @parse_result = Prism.parse_lex(@source)
```

Comment:
should we rename?
```suggestion
      @parse_lex_result = Prism.parse_lex(@source)
```
(few other places too)

Reply:
I'm not sure we'll get much value out of renaming it. Both regular parse and parse_lex return a parse result.